introduction_to_amazon_algorithms/jumpstart-foundation-models/code-llama-fine-tuning-evaluate-human-eval.ipynb

```diff
@@ -0,0 +1,1284 @@
+{
```

Comment:
<p>question: do we have an eventual goal to transition this to use clarify via fmeval? <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/clarify-foundation-model-evaluate-auto-tutorial.html" target="_blank">https://docs.aws.amazon.com/sagemaker/latest/dg/clarify-foundation-model-evaluate-auto-tutorial.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>